src/message/header/content_type.rs

```diff
@@ -9,9 +9,13 @@ use mime::Mime;
 use super::{Header, HeaderName};
 use crate::BoxError;
 
-/// `Content-Type` of the body
+/// This struct can represent any valid [mime type] by using its `parse`
+/// function! There are just these two constant values `TEXT_PLAIN` and
+/// `TEXT_HTML` because they are the most-used mime-types.
```

Comment:
Sorry for the delay on the review. I think this sounds better

```suggestion
/// `Content-Type` of the body
///
/// This struct can represent any valid [mime type], which can be parsed via
/// [`ContentType::parse`]. Constants are provided for the most-used mime-types.
```

Reply:
Looks good to me! What do you think about adding some links as well? Like this:

```rust
/// `Content-Type` of the body
///
/// This struct can represent any valid [mime type], which can be parsed via
/// [`ContentType::parse`]. Constants are provided for the most-used mime-types.
///
/// [mime type]: https://docs.rs/mime/0.3.16/mime/index.html#constants
```

Reply:
`[mime_type]` is already linked below :smiley:

Reply:
Woops, than I can't complain ;)

Reply:
Do I have to change it in my commit or is this changed already "stored"? The way how you changed the commit is new for me and I don't really know how it works :eyes:

Reply:
This is a [suggested change](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change). If you'd like to accept it as-is there's a "Commit suggestion" button in the GitHub Desktop UI which will commit the change for you.

Reply:
Ah I see, thanks!